README.md

```diff
@@ -52,7 +53,7 @@ Builds for local instances:
 flutter build linux lib/main_local.dart
 ```
 
-## Signing Android AppBundle for Google Play upload
+### Signing Android AppBundle for Google Play upload
```

Comment:
Can this section be moved to `doc` directory instead?

Reply:
Yes, I will do

Reply:
I made changes, however README.md I plan to change more with another PR